src/traces/bar/attributes.js

```diff
@@ -59,6 +60,7 @@ module.exports = {
 
     text: scatterAttrs.text,
     hovertext: scatterAttrs.hovertext,
+    hovertemplate: hovertemplateAttrs(),
```

Comment:
We should append the `hovertemplate` descriptions with a list of flags (i.e. event data keys) available.

Reply:
On a related note: I like that for some traces, like `pie`, we test that event data has the correct keys:
https://github.com/plotly/plotly.js/blob/ad937d20a83fb3d183916e715c1a3aac6cef5921/test/jasmine/tests/pie_test.js#L824-L856

We could potentially use the same array of field names for the test and for the description making sure we have one source of truth :thinking:. This improvement could be done in another PR.

Reply:
> We could potentially use the same array of field names for the test and for the description making sure we have one source of truth

Yeah, good call. We usually put things like that in "constants" files like this one:

https://github.com/plotly/plotly.js/blob/master/src/traces/scatter/constants.js

> This improvement could be done in another PR.

That's up to you!